docs/cli-roadmap.md

```diff
@@ -0,0 +1,82 @@
+# Kubernetes CLI/Configuration Roadmap
+
+See also issues with the following labels:
+* [area/config-deployment](https://github.com/GoogleCloudPlatform/kubernetes/labels/area%2Fconfig-deployment)
+* [component/CLI](https://github.com/GoogleCloudPlatform/kubernetes/labels/component%2FCLI)
+* [component/client](https://github.com/GoogleCloudPlatform/kubernetes/labels/component%2Fclient)
+
+At least the first three issues should block 1.0.
+
+1. stop command in kubectl [#2144](https://github.com/GoogleCloudPlatform/kubernetes/issues/2144): Gracefully terminate any object(s), to the extent currently possible. We also eventually want first-class support in the API for graceful termination. [#1535](https://github.com/GoogleCloudPlatform/kubernetes/issues/1535)
+1. Kind-based filtering on object streams -- only operate on the kinds of objects specified
```

Comment:
Should this really be a blocker for 1.0? The only reason I can think why this would be a must-have is for scalability, but do we expect to support clusters large enough for this to matter in 1.0? Or maybe this is just to filter out Events, which I guess are a disproportionate amount of the object update traffic?